freesound/settings.py

```diff
@@ -275,6 +281,10 @@
 MIRROR_ANALYSIS = None  # list of locations to mirror contents of SOUNDS_PATH, set to None to turn off
 MIRROR_AVATARS = None  # list of locations to mirror contents of AVATARS_PATH, set to None to turn off
 
+# Footer banner
+FOOTER_BANNER_CONTENTS = \
+    '<a href="javascript:void(0);" onclick="openSurveyPage();hideFooterBanner();setSurveyVisited();">' \
+    '<img src="/media/images/content/freesound-survey-2017.png"/></a>'  # Set to None for no banner
```

Comment:
yeah that's all my fault I know ;)
maybe we should have waited for my first pull request to be approved before building more stuff on top of it. I don't know, I guess I thought it was the best place to put that but I also felt a bit weird. Any suggestion where should we place that?